README.md

```diff
@@ -158,18 +152,23 @@ a sub-directory name found in `agent/bench-scripts/tests`. For example:
 The first runs all the `pbench-fio` tests, while the second runs all the
 `pbench-uperf` and `pbench-linpack` tests.
 
+You can run the `build.sh` script to execute the linters, to run the unit tests
```

Comment:
Dang GitHub won't let me comment where I'd like: but some whim caused me to expand above this diff and I found several `tox` references to the server `legacy` tests up there. If you feel like it, you might nuke them while you're here.